weave

```diff
@@ -1553,25 +1550,10 @@ EOF
     launch)
         deprecation_warnings "$@"
         check_not_running router $CONTAINER_NAME        $BASE_IMAGE
-        check_not_running proxy  $PROXY_CONTAINER_NAME  $BASE_EXEC_IMAGE
         check_not_running_plugin_v2
         COMMON_ARGS=$(common_launch_args "$@")
-        launch_router --plugin "$@"
-        launch_proxy  $COMMON_ARGS
-        ;;
-    launch-router)
-        deprecation_warnings "$@"
-        check_not_running router $CONTAINER_NAME $BASE_IMAGE
-        check_not_running_plugin_v2
-        launch_router "$@"
-        echo $ROUTER_CONTAINER
-        ;;
-    launch-proxy)
-        deprecation_warnings "$@"
-        check_not_running proxy $PROXY_CONTAINER_NAME $BASE_EXEC_IMAGE
-        check_not_running_plugin_v2
-        launch_proxy "$@"
-        echo $PROXY_CONTAINER
+        launch --plugin "$COMMON_ARGS" "$@"
```

Comment:
As discussed offline, there really are two "APIs" here:
- the "API" exposed by the `weave` script;
- the "API" exposed by the `weaver` container.

It has been agreed that:
1. Go code should have plugin and proxy disabled by default. 
    - This is leaner by default. 
    - This makes calling `weaver` slightly nicer in the case of the `weave-kube`'s `launch.sh` script or for third parties like Cloud66.
2. `--proxy`/`--plugin` should be passed in by `weave launch` to continue to start all three components by default and have a consistent behaviour with what was there previously.
3. It should be possible to pass `--x=false` (`x = { proxy, plugin }`), which `mflag.BoolVar(&x, []string{"-x"}, false, "...")` would understand, and given the last value takes precedence, `--x --x=false` would lead to `x` being disabled.